### PR TITLE
Some opam updates

### PR DESCRIPTION
**Pull request by rjbou**
on syntax, plugin & file detection

### CODE REVIEW

File: plugin/opam.vim [hunk 1]
```diff
@@ -69,9 +69,9 @@ function! s:Opam(bang,...) abort
 endfunction
 
 function! s:Complete(A,L,P)
-  let installed = split((system("opam switch -s -i 2> /dev/null")), "\n")
-  call map(installed, 'opam#chomp(v:val)')
-  return join(installed, "\n")
+  let switches = split((system("opam switch --short 2> /dev/null")), "\n")
```
Comment (rjbou) [inline plugin/opam.vim:72]: `-i` i no more available, is it required to have only installed ones?

File: syntax/opam.vim [hunk 2]
```diff
@@ -11,8 +11,36 @@ endif
...
+syn keyword opamKeyword1 pin-depends
+syn keyword opamKeyword1 remove
+syn keyword opamKeyword1 run-test
+syn keyword opamKeyword1 setenv
+syn keyword opamKeyword1 substs
+syn keyword opamKeyword1 synopsis
+syn keyword opamKeyword1 syntax
+syn keyword opamKeyword1 tags
+syn keyword opamKeyword1 version
+syn match opamKeyword2 "\v(bug-reports|post-messages|opam-version|dev-repo|build)"
```
Comment (rjbou) [inline syntax/opam.vim:43]: What is the reason to have both `opamKeyword1` and `opamKeyword2` ?

File: plugin/opam.vim (hunk 1 above)
Comment (rgrinberg) [reply plugin/opam.vim:72]: i don't think so

File: syntax/opam.vim (hunk 2 above)
Comment (rgrinberg) [reply syntax/opam.vim:43]: No idea. You can try including it in opamKeyword1 and see if it works.

**Pull request merged**